arcus_orientation.md

```diff
@@ -793,28 +801,28 @@ When installation is complete, click "OK" to close the installation window.
 You can now return to R and your packages will be available. 
 You can load them with `library()` or `require()` as you normally would.
 
-Note that if you forget and attempt to run `install.packages()` when you have internet disabled, you should see a helpful error message reminding you to use the install form instead. 
+Note that if you forget and attempt to run `install.packages()` in Internet Disabled mode, you should see a helpful error message reminding you to use the Install Tool instead. 
 
 ![A screenshot of RStudio's console pane, where the command "install.packages('ggplot2')" is followed by the error message "Internet connection is Disabled. Installing packages with install.packages() from within an Arcus lab will not work. Instead, use the install form available in the Tools section of your lab dashboard. For help, submit a ticket at https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/303"](media/install_r_error_msg.png)
```

Comment:
This screenshot also needs to be updated, right? (provided that there are plans to edit the code of the package so that the error message that it displays refers to it as the Install Tool rather than form)